cli/kaos_cli/utils/decorators.py

```diff
@@ -67,17 +67,32 @@ def wrapper(*args, **kwargs):
         provider = kwargs["cloud"]
 
         if provider == "DOCKER":
+
+            # Docker Desktop is running WITH single-node kubernetes cluster
             cmd = "kubectl get services --context docker-for-desktop"
             exitcode, out, err = run_cmd(cmd)
-            if "command not found" in str(err):
-                click.echo("{} - Docker Desktop not found. Please install any one of them and then retry build".format(
-                    click.style("Warning", bold=True, fg='yellow')))
+            error_codes = {"Unable to connect to the server",
+                           "did you specify the right host or port?"}
+            if any([e in str(err) for e in error_codes]):
+                click.echo(
+                    "{} - Docker Desktop with Kubernetes is currently {}\n\n"
+                    "Please {} Docker Desktop and {} Kubernetes".format(
+                        click.style("Warning", bold=True, fg='yellow'),
+                        click.style("disabled", bold=True, fg='red'),
+                        click.style("start", bold=True, fg='green'),
+                        click.style("enable", bold=True, fg='green')))
                 sys.exit(1)
-            if "Unable to connect to the server" in str(err):
+
+            # Docker Desktop context is set
+            cmd = "kubectl config current-context"
+            exitcode, out, err = run_cmd(cmd)
+            if "docker-desktop" not in str(out):
```

Comment:
There is some inconsistency regarding context naming in docker. The name of the local context can be "docker-desktop" or "docker-for-desktop", both seem to be valid. 
<img width="1205" alt="Screenshot 2019-11-12 at 10 30 38" src="https://user-images.githubusercontent.com/10234643/68659468-93d87c80-0537-11ea-86b4-7e0c7ee02a15.png">

Reply:
will adjust. thanks for the info